csrc/transform_rfactor.cpp

```diff
@@ -417,25 +449,9 @@ std::pair<TensorDomain*, TensorDomain*> TransformRFactor::runReplay(
     }
   }
 
-  // Axes in the original_td that are in the history of the rfactored domains.
-  // These will mark which iter domains must be preserved as static
-  // transformations to preserve compute semantics.
-  auto all_deps_of_logical = DependencyCheck::getAllValsBetween(
```

Comment:
code movement: moved to `ReplayRFactor` constructor. We compute vals between `maybeRoot/logical/allocation` to `rfactor_axes`.